api/src/lib/auth.js

```diff
@@ -7,9 +7,23 @@
 
 // @ts-check
 
-import { AuthenticationError } from '@redwoodjs/api'
+import { AuthenticationError } from '@redwoodjs/graphql-server'
 import { logger } from './logger'
 
+import admin from 'firebase-admin'
+
+const config = {
+  apiKey: process.env.FIREBASE_API_KEY,
+  authDomain: process.env.FIREBASE_AUTH_DOMAIN,
+  databaseURL: process.env.FIREBASE_DATABASE_URL,
+  projectId: process.env.FIREBASE_PROJECT_ID,
+  storageBucket: process.env.FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.FIREBASE_MESSAGING_SENDER_ID,
+  appId: process.env.FIREBASE_APP_ID,
+}
+
+admin.initializeApp(config)
+
```

Comment:
Not sure where to put the api-side firebase-admin initialization.  Without this, the graphql endpoint currentUser handler will not work, because the token decoder uses firebase's verifyIdToken() to decode the JWT and verify it.

Reply:
By any chance @dthyresson or @dac09 have some input here? 

I'm upgrading the playground auth from `0.36.0` to `0.37.0` and firebase seems broken. Maybe we could sort this PR out instead of a separate upgrade PR.

Reply:
Ok, I upgraded the redwood dependencies in this PR to 0.37.0, had to add a api/src/directives as well and it seems to be working for me.